package.json

```diff
@@ -18,7 +19,6 @@
   },
   "devDependencies": {
     "browserify": "^11.0.1",
```

Comment:
> though it's a bit odd browserify is a dev dependency and not production dependency

Moving it to dependencies would have no effect, though, on the end-users bundling CommonJS with browserify.

Worth noting:
Using browserify + package.json transforms does lead to a "peer dependency" on browserify, as opposed to Webpack or another bundler. This only affects those using the CommonJS export of AFrame.